go/obsclient/obsclient.go

```diff
@@ -76,7 +76,7 @@ func (oc *ObsClient) RollupHeaderByHash(hash gethcommon.Hash) (*common.Header, e
 
 // Health returns the health of the node.
 func (oc *ObsClient) Health() (bool, error) {
-	var healthy *bool
+	var healthy *hostcommon.HealthCheck
 	err := oc.rpcClient.Call(&healthy, rpc.Health)
-	return *healthy, err
+	return (*healthy).OverallHealth, err
```

Comment:
Do you need the brackets and *operator here? I'd expect `return healthy.OverallHealth` to work because of go magic.

Reply:
Good catch 👍